agent/engine/docker_task_engine.go

```diff
@@ -975,6 +975,7 @@ func (engine *DockerTaskEngine) AddTask(task *apitask.Task) {
 				return
 			}
 			engine.AddTask(engine.serviceconnectRelay)
+			logger.Info("docker_task_engine: Added AppNet Relay task to engine")
```

Comment:
A small Q. should we log the task ID if it could help debug?
```
logger.Info("Added AppNet Relay task to engine for the task", logger.Fields{
    "field.TaskID": task.GetID(),
    })
})
```

Reply:
Sure thing. Will batch with the next PR.